documentation/src/main/asciidoc/topics/con_side_caching.adoc

```diff
@@ -0,0 +1,22 @@
+[id='side-caching_{context}']
+= Side caching
+
+{brandname} stores data that applications retrieve from persistent storage, which reduces the number of read operations to persistent storage and increases response times for subsequent reads.
+
+.Side caches
+//Community content
+ifdef::community[]
+image::side-cache-ispn.png[This illustration depicts a {brandname} side cache to which applications write to avoid repeatedly reading the same entries from persistent storage.]
```

Comment:
Wondering what is your opinion about:
* stage 1 - > add an arrow from the cache to  the application saying "not found"
* stage 2 -> split stage 2 into two stages: the first one reads from persistence, the second one stores in cache

Reply:
Yeah, we could do that. I'll need to work with the illustrator so would like to do that in a separate PR. I've created https://issues.redhat.com/browse/ISPN-13701 so we don't lose track of it.